<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>576</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1024</width>
    <height>576</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="_central_widget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="GraphicsArea" name="_graphics_view">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="_horizontal_layout" stretch="1,1,1">
      <item>
       <widget class="QLabel" name="_score_label">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_retry_button">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Retry</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_pause_button">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Pause</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,1">
      <item>
       <widget class="QCheckBox" name="_check_box">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>7</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string> can you lose?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="_speed_slider">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="_mob_slider">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>99</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="_control_layout"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphicsArea</class>
   <extends>QGraphicsView</extends>
   <header location="global">graphicsarea.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>_pause_button</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>Pause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>696</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>677</x>
     <y>454</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_retry_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>Retry()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_mob_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>DifficultyChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>575</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>796</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_check_box</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>LosableChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>16</x>
     <y>452</y>
    </hint>
    <hint type="destinationlabel">
     <x>-5</x>
     <y>456</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_speed_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>SpeedChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>480</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>473</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Pause()</slot>
  <slot>Retry()</slot>
  <slot>LosableChanged()</slot>
  <slot>DifficultyChanged()</slot>
  <slot>SpeedChanged()</slot>
 </slots>
</ui>
